src/xrEngine/Text_Console.cpp

```diff
@@ -296,9 +296,9 @@ void CTextConsole::DrawLog(HDC hDC, RECT* pRect)
         COLORREF c2 = (COLORREF)bgr2rgb(get_mark_color(cm));
         SetTextColor(hDC, c2);
         u8 b = (is_mark(cm)) ? 2 : 0;
-        LPCSTR pOut = ls + b;
+        pcstr pOut = ls + b;
 
-        BOOL res = TextOut(hDC, 10, ypos, pOut, xr_strlen(pOut));
+        bool res = TextOut(hDC, 10, ypos, pOut, xr_strlen(pOut));
```

Comment:
Just in case, use `!!` to remove possible compiler warnings.
```suggestion
        bool res = !!TextOut(hDC, 10, ypos, pOut, xr_strlen(pOut));
```